Extensions/Today/TodayUX.swift

```diff
@@ -28,3 +28,19 @@ struct TodayUX {
         }
     }
 }
+// This class with the function height is to dynamically calculate the height for a label and set the constraints accordingly, the height function takes some text and the text style size category and insert it into a label that dynamically resizes itself and get the frame of the resized label and consequently the right height to use.
+class DynamicLabelResize {
```

Comment:
@noorhashem would it be possible to have this method as a part of UILabel Extension?

Reply:
Added a follow-up comment as we won't be using this method.